lib/views/login_signup.dart

```diff
@@ -0,0 +1,539 @@
+import 'package:flutter/services.dart';
+
+class login_signup extends StatefulWidget{
+  login_signup_state createState() => login_signup_state();
+}
+
+class login_signup_state extends State<login_signup>{
+
+  TextEditingController _txt1 = TextEditingController();
+  TextEditingController _txt2 = TextEditingController();
+  TextEditingController _username = TextEditingController();
+  TextEditingController _password = TextEditingController();
+  TextEditingController _email = TextEditingController();
+  TextEditingController _contact = TextEditingController();
+  FocusNode fnode1 = FocusNode();
+  FocusNode fnode2 = FocusNode();
+  FocusNode fnode3 = FocusNode();
+  FocusNode uname = FocusNode();
+  FocusNode password = FocusNode();
+  FocusNode email = FocusNode();
+  FocusNode contact = FocusNode();
+  final _formKey = GlobalKey<FormState>();
+  final _formKey2 = GlobalKey<FormState>();
+  late double h1,h2,h3; // this will be used for setting margin over waves
+  bool first_pane= true,onstart=true,display_text=true;// true if first page is open, false if close. onstart is only set once
+  bool visibility = true,visibility2 = true;
+
+  String? _validateEmail(String? value){
+    if(value!.isEmpty){
+      return "Please enter an email id";
+    }
+    String email_format = "[a-zA-Z0-9\+\.\_\%\-\+]{1,256}"+          //Regular Expression to match the email id
+        "\\@""[a-zA-Z0-9\\-]{0,64}"+
+        "(""\\.""[a-zA-Z0-9][a-zA-Z0-9\\-]{0,25}"+")+";
+    RegExp regEx = new RegExp(email_format);
+
+    if(regEx.hasMatch(value)){
+        return null;
+    }
+    return "Please Enter a valid Email-id";
+  }
+
+  void _pass_vis() {
+    setState(() {
+      visibility = !visibility;
+    });
+  }
+
+  void _pass_vis2() {
+    setState(() {
+      visibility2 = !visibility2;
+    });
+  }
+
+  // This method will change the size of containers to show different page
+
+  void change_size(){
+    setState(() {
+      if(first_pane){
+        first_pane = false;
+        h1 = MediaQuery.of(context).size.height*0.10;
+        h2 = MediaQuery.of(context).size.height*0.12;
+        h3 = MediaQuery.of(context).size.height;
+        display_text = false;
+      }else{
+        first_pane = true;
+        h1 = MediaQuery.of(context).size.height*0.3;
+        h2 = MediaQuery.of(context).size.height*0.318;
+        h3 = MediaQuery.of(context).size.height*0.33;
+        display_text = true;
+      }
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    if(onstart){
+      onstart = false;
+      h1 = MediaQuery.of(context).size.height*0.3;
+      h2 = MediaQuery.of(context).size.height*0.318;
+      h3 = MediaQuery.of(context).size.height*0.33;
+    }
+    return Scaffold(
+        resizeToAvoidBottomInset: false,
+      body: Stack(
+        children: [
+          SizedBox(
+              height: double.infinity,
+              width: double.infinity,
+              child: Image.asset("lib/assets/mountain.jpg",fit: BoxFit.fill,)
```

Comment:
Do not use the image path directly here. Instead, you can create a separate class declare path there and use the variable here.